js/ui/overlay/ui.overlay.js

```diff
@@ -996,7 +992,7 @@ const Overlay = Widget.inherit({
     },
 
     _getDragTarget: function() {
-        return this.$content();
+        return this._$content;
```

Comment:
Seems like this change can break drag in popup

Reply:
Done